### PR TITLE
Update EntEx landing page info

### PR DESCRIPTION
**Pull request by srpearce**
@mattdrayer Please review for technical accuracy.

@mhoeber @lamagnifica @catong Can one of you review this one?

### CODE REVIEW

**Comment by mhoeber** [comment]
I will review

File: en_us/shared/building_and_running_chapters/building_course/provide_overview.rst [hunk 1]
```diff
@@ -273,23 +271,8 @@ students have a positive experience with entrance exams.
...
-
-  * After students complete the entrance exam, they have to click the **Entrance
-    Exam** tab again or refresh the page in their browser. When the page
-    refreshes, students can see all currently available course content.
-
-    .. image:: ../../../shared/building_and_running_chapters/Images/EntEx_CourseAccordionAfterPass.png
-      :width: 500
-      :alt: The student view after the student has passed the entrance exam,
-          with all available course sections listed in the course accordion
+* On your course About page or in a course e-mail, include a link to the
```
Comment (mhoeber) [inline en_us/shared/building_and_running_chapters/building_course/provide_overview.rst:274]: students won't see the about page until they are done though, right?

File: en_us/students/source/SFD_prerequisites.rst [hunk 2]
```diff
@@ -52,29 +52,29 @@ If you enroll in a course that requires an entrance exam, the course appears on
...
   :width: 500
-  :alt: The Course Updates & News page with the Entrance Exam tab circled on the
-      left
+  :alt: The Entrance Exam page with the first problem visible
 
-To begin the entrance exam, select the **Entrance Exam** tab.
+After you answer the questions in the entrance exam, refresh the page. The
+message at the top of the page lets you know if you have passed the exam. If
+you refresh the page before you answer all the questions in the exam, you can
+see your current score at the top of the page.
```
Comment (mhoeber) [inline en_us/students/source/SFD_prerequisites.rst]: do they really need to refresh the page?  Isn't there a check button?

**Comment by mhoeber** [comment]
@srpearce  looks good, just asked a couple of questions

File: en_us/shared/building_and_running_chapters/building_course/provide_overview.rst (hunk 1 above)
Comment (srpearce) [inline en_us/shared/building_and_running_chapters/building_course/provide_overview.rst:274]: The About page is the page on the Drupal site that students see before they register - you might be thinking of the **Course Info** tab, which is in the LMS.

File: en_us/students/source/SFD_prerequisites.rst (hunk 2 above)
Comment (srpearce) [inline en_us/students/source/SFD_prerequisites.rst]: Yep, at the moment students have to refresh the page after they click **Check** for the score to update.

**Comment by srpearce** [comment]
@jhendersonedx Sorry, forgot to tag you for review earlier.

**Comment by marcotuts** [comment]
JIRA is warning that this PR is still open though the stories have now merged. No rush here, just wanted to point out the JIRA warning. (warning is here: https://openedx.atlassian.net/projects/SOL/versions/11601)

**Comment by srpearce** [comment]
@marcotuts Thanks. I think the changes that this update describes (students go directly to the entrance exam) are going out in this week's release, correct? If that's the case, I'll merge this and rebuild the docs as soon as the release is live.

**Pull request merged**